bin/resty

```diff
@@ -172,10 +191,13 @@ my $loader = <<_EOC_;
             local gen
             do
                 $args
-                local fname = $quoted_luafile
-                local f = assert(io.open(fname, "r"))
-                local chunk = f:read("*a")
-                gen = assert(loadstring(chunk, "$chunk_name"))
+                $inline_lua
+                $file_lua
```

Comment:
We should avoid indentation for these two interpolated variables here since their values already have indentation.